webassembly-language-support.md

```diff
@@ -70,9 +87,9 @@ For each environment, we use the following icons to indicate a level of support:
 - ✅  Usable
 - ⏳ In progress
 - ❌ Not implemented
-- N/A Not applicable
+- `N/A` Not applicable
 
-The Fermyon Platform requires [WASI](https://wasi.dev) support. Any language that has a ✅ for WASI should be supported on the Fermyon Platform.
+Spin, Fermyon Platform, Fermyon Cloud require [WASI](https://wasi.dev) support. Any language that has a ✅ for WASI should be supported on the Fermyon Platform. The *Spin SDK* indicates that there is additional libraries available for Spin.
```

Comment:
```suggestion
Spin, Fermyon Platform and Fermyon Cloud require [WASI](https://wasi.dev) support. Any language that has a ✅ for WASI should be supported on the Fermyon Platform. The *Spin SDK* indicates that there are additional libraries available for Spin.
```

Reply:
Oops... didn't see this before my rebase. Added this manually.